### PR TITLE
Add new tests for sensitivities with empty sens_ind / plist

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: tests/testModels.m [hunk 1]
```diff
@@ -23,6 +26,10 @@ function testModels()
             model_rtol = 1e-5;
         end
         for itest = 1:length(info.Groups(imodel).Groups)
+            if(any(strcmp(ignoredTests, info.Groups(imodel).Groups(itest).Name)))
```
Comment (FFroehlich) [inline tests/testModels.m]: `ismember(info.Groups(imodel).Groups(itest).Name, ignoredTests)` would be more consistent with the rest of the script.
